tests/test_e2e.py

```
@@ -1254,3 +1256,240 @@ def test_list_re_replace():
+    except ResponseError as e:
+        assert "Received 2 arguments to function 'ceil', expected at most 1" in str(e)
+
+
+def test_e():
+    res = query("RETURN e() AS name")
+    assert res.result_set == [[2.71828182845905e0]]
+
+    # wrong number of args
+    try:
+        query("RETURN e(1) AS name")
+        assert False, "Expected an error"
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)